perf/specjbb/run/windows_scripts/README.md

```diff
@@ -0,0 +1,51 @@
+# SPECjbb2015 Automation Scripts Guide
+
+## Overview
+
+This guide encompasses information on the use of automation scripts designed for the SPECjbb2015 benchmark, facilitating both Composite and MultiJVM modes. The scripts aim to simplify the execution process, enabling users to run the benchmark with customized settings through external configuration files.
+
+## Available Scripts
+
+1. Composite Mode Script (`run_composite.bat`): Automates the benchmark execution in Composite mode, where the Controller, TxInjector, and Backend are launched within a single JVM.
+
+2. MultiJVM Mode Script (`run_multijvm.bat`): Facilitates running the Controller, TxInjectors, and Backends in separate JVMs on the same server, ideal for testing distributed setups or when isolating components is required.

```

Comment:
Probably not strictly true that it can test distributed setups, I think you need the distributed scripts for that?